src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -106,27 +105,45 @@
   (ct/report reports))
 
 (defmacro checking
-  "A macro intended to replace the testing macro in clojure.test with a
+  {:doc      "A macro intended to replace the testing macro in clojure.test with a
   generative form. To make (testing \"doubling\" (is (= (* 2 2) (+ 2 2))))
   generative, you simply have to change it to
-  (checking \"doubling\" 100 [x gen/int] (is (= (* 2 x) (+ x x)))).
+  (checking \"doubling\" [x gen/int] (is (= (* 2 x) (+ x x)))).
 
-  You can optionally pass in a map options instead of the number of tests,
+  You can optionally pass in a number of a map options,
```

Comment:
"a number or a map of options"

you could perhaps also add "similar to defspec"